src/script/processTarget/setWebpack/setEnv.js

```diff
@@ -0,0 +1,4 @@
+module.exports = (path) => {
+  process.env.NODE_ENV = 'development'
+  process.env.NODE_PATH = path
```

Comment:
I think we don't need to do that anymore, because in webpack config we can add node_modules fallback in `[ override.js](https://github.com/Zenika/react-workbench/blob/0fa47c9aa7cbe18f47446099a0a30ae4abb67422/src/script/processTarget/setWebpack/override.js#L24)`
After a quick test it seems working.
Moreover, by setting `NODE_PATH` on workbench, we can user absolute import.
what do you think ?